localization_node:
  ros__parameters:
    cam_params:
      size: [4608, 2592]          # Camera resolution [width, height]
      camera_height: 0.63         # Height of the camera from the ground in meters
      timer_period: 0.5           # Timer period in seconds for frame processing
      aruco_square_size: 0.036    # Size of the ArUco marker in meters
      fx: 2078                   # Focal length in pixels (x-axis)
      fy: 2068                   # Focal length in pixels (y-axis)
      angle_offset: 1.5708        # Angle offset in radians

    aruco_params:
      11:
        t_x: 4.352
        t_y: 6.368
        yaw: 0.0
      618:
        t_x: 4.402
        t_y: 7.358
        yaw: 0.0
      645:
        t_x: 0.0
        t_y: 0.0
        yaw: 0.0
      614:
        t_x: 4.38
        t_y: 4.56
        yaw: 0.0
      914:
        t_x: 1.0
        t_y: 1.0
        yaw: 0.0
